pages/index.js

```diff
@@ -308,6 +309,10 @@ const Logos = [
   >
     <CoinbaseLogo />
   </CompanyLogo>,
+  
+  <CompanyLogo height="2rem" key="https://culturetrip.com" similarWebGlobalRank={1487275}>
```

Comment:
The [similar web global rank for `theculturetrip.com` (which is the actual domain) is `4,945`](https://www.similarweb.com/fr/website/theculturetrip.com)!

```suggestion
  <CompanyLogo height="2rem" key="https://theculturetrip.com" similarWebGlobalRank={4945}>
```